docs/internal/contributing/how-integration-tests-work.md

```diff
@@ -1,4 +1,4 @@
-Mimir integration tests are written in Go and based on a [custom framework](https://github.com/grafana/mimir/tree/main/integration/e2e) running Mimir and its dependencies in Docker containers and using the Go [`testing`](https://golang.org/pkg/testing/) package for assertions. Integration tests run in CI for every PR, and can be easily executed locally during development (it just requires Docker).
+Mimir integration tests are written in Go and based on a [custom framework](https://github.com/grafana/mimir/tree/main/integration/e2emimir) running Mimir and its dependencies in Docker containers and using the Go [`testing`](https://golang.org/pkg/testing/) package for assertions. Integration tests run in CI for every PR, and can be easily executed locally during development (it just requires Docker).
```

Comment:
Ehm... the framework is actually at https://github.com/grafana/e2e. @hi-rustin do you mind opening another PR to fix it, please? :)

Reply:
🤦 At least the link works now :)